src/extras/MovieClip.js

```diff
@@ -56,13 +56,13 @@ function MovieClip(textures)
     this.onComplete = null;
 
     /**
-     * The MovieClips current frame index (this may not have to be a whole number)
+     * Elapsed time since animation has been started, used internally to display current texture
```

Comment:
@drkibitz . I know this convention you mentioned but I meant I wasn't sure if this var should be private or not:)   ok, i However  I've rethought and agree with you, it makes sense to make it privete,  will change it soon